src/Administration/Resources/app/administration/src/app/component/entity/sw-one-to-many-grid/index.js

```diff
@@ -63,18 +63,29 @@ Component.extend('sw-one-to-many-grid', 'sw-data-grid', {
             page: 1,
             limit: 25,
             total: 0,
-            intial: true,
+            initial: true,
```

Comment:
I think so. Either you declare a new variable to replace it, or you keep it as is.

Reply:
Would you guys like to add a deprecated for this variable? It's the wrong typo now.

Reply:
Would this work?

```suggestion
            /** @deprecated...*/
            intial: true,
            initial: this.intial,
```

Reply:
It does not work when generating items. Can we do it?
/* @deprecated: tag:v6.7.0 - Will be removed because wrong typo, use initial instead */
 intial: true,
 initial: true,

Reply:
it'd be fine, in the createdComponent you could have something like this

```js
this.initial = this.intial
```